browser-extension/build.sh

```diff
@@ -1,5 +1,18 @@
 #!/usr/bin/env bash
+
+#
+# Build a .zip target that can be uploaded to:
+# - Chrome Web Store
+# - Firefox Add-ons
+#
+
 set -eu
 
-cd src
-zip -r -FS ../tournesol_extension.zip *
+TARGET='tournesol_extension.zip'
+
+pushd src
+zip -r -FS ../${TARGET} *
+popd
+
+zip ${TARGET} LICENSE
```

Comment:
The script relies on being run from the "browser-extension" directory. 
Not really a problem, it was already the case before. But using the script path as the reference in these commands could be a slight improvement (similar to infra scripts).